projects/canopy/src/lib/tabs/docs/guide.stories.mdx

```diff
@@ -10,12 +10,12 @@ import Feedback from '../../docs/common/feedback.md?raw';
 
 ## Usage
 
-Import the tabs module in your application:
+Import the tabs components in your application:
 
 ```js
 @NgModule({
   ...
-  imports: [ ..., LgTabsModule ],
+  imports: [ ..., LgTabsComponent, etc. ],
```

Comment:
What does etc refer to? Also, shall we add a usage part for standalone components?